diff -rupN gen-java/org/styloot/maryjane/gen/MaryJane.java gen-java-new/org/styloot/maryjane/gen/MaryJane.java
--- gen-java/org/styloot/maryjane/gen/MaryJane.java	2011-04-14 13:15:02.198459844 -0400
+++ gen-java/org/styloot/maryjane/gen/MaryJane.java	2011-04-14 13:10:34.437459939 -0400
@@ -17,8 +17,6 @@ import java.util.Collections;
 import java.util.BitSet;
 import java.nio.ByteBuffer;
 import java.util.Arrays;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;

 import org.apache.thrift.*;
 import org.apache.thrift.async.*;
@@ -240,7 +238,6 @@ public class MaryJane {
   }

   public static class Processor implements TProcessor {
-    private static final Logger LOGGER = LoggerFactory.getLogger(Processor.class.getName());
     public Processor(Iface iface)
     {
       iface_ = iface;
